migration/V5_sni_hostname_migration.go

```diff
@@ -0,0 +1,30 @@
+package migration
+
+import (
+	"code.cloudfoundry.org/routing-api/db"
+	"code.cloudfoundry.org/routing-api/models"
+)
+
+type V5SniHostnameMigration struct{}
+
+var _ Migration = new(V5SniHostnameMigration)
+
+func NewV5SniHostnameMigration() *V5SniHostnameMigration {
+	return &V5SniHostnameMigration{}
+}
+
+func (v *V5SniHostnameMigration) Version() int {
+	return 5
+}
+
+func (v *V5SniHostnameMigration) Run(sqlDB *db.SqlDB) error {
+	_, err := sqlDB.Client.Model(&models.TcpRouteMapping{}).RemoveIndex("idx_tcp_route")
```

Comment:
Is this because the table schema changes? Should we add the index back when we are done?

Reply:
The next line that does auto-migration takes care of that. If the index is there it doesn't appear to migrate the index. In V4 they delete and add the index manually but I found you can delete it and then let auto-migration add it back.